dbt/macros/tests/test_all.sql

```diff
@@ -5,6 +5,5 @@
     {% do test_generate_schema_name() %}
     {% do test_generate_alias_name() %}
     {% do test_format_additional_select_columns() %}
-    {% do test_pre_stage_filters() %}
```

Comment:
These tests weren't really useful in the first place, and are even less useful now that our two pre-stage macros don't even share common logic. I got rid of the tests to reduce the future maintenance burden for this view.